app/presenters/edition_presenter.rb

```diff
@@ -34,7 +33,7 @@ def for_content_store(payload_version)
     end
 
     def for_message_queue(payload_version)
-      present.merge(
+      present.except(:auth_bypass_ids).merge(
```

Comment:
I can't see a test for the absence of `auth_bypass_ids` in the message queue (unless I'm misunderstanding something) - can you create one?

Reply:
I've removed this `except` in favour of adding the field to NON_PRESENTED_PROPERTIES, the logic is still the same however so i've added a test here - https://github.com/alphagov/publishing-api/pull/1625/files#diff-8180fe08951fde62fc6dd9c41bee0d17R60-R62.